src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,36 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
```

Comment:
Please destruct your post object. It will be more clear

Reply:
Why it will be clearer, if instead of `post.title` we will have just `title`?